src/models/Group.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require("mongoose");
+
+const groupSchema = new mongoose.Schema(
+  {
+    name: {
+      type: String,
+      required: [true, "please provide a name!"],
+    },
+    host: {
+      type: String,
+      required: [true, "a group must have a host!"],
+      unique: true,
+    },
+    members: [{ type: mongoose.Schema.Types.ObjectId, ref: "User" }],
```

Comment:
I'm new to mongo/mongoose...if we want to display the list of groups a specific user belongs to, should we copy the data on the user model? So should we have an array of groups on user?

Reply:
This is just a preliminary setup, I am currently testing out all these schemas. So the way you are currently describing it might not work with the current setup in here, and yes there should be a reference of the groups that the user has joined. I'll make some modification.